src/controllers/CheckYourAppealController.ts

```diff
@@ -26,7 +34,38 @@ export class CheckYourAppealController extends BaseAsyncHttpController {
             .orDefault({});
 
         return this.render('check-your-appeal', { ...appealsData, userProfile });
+    }
+
+    @httpPost('')
+    public async handleFormSubmission(req: Request): Promise<HttpResponseMessage> {
+        const userProfile = req.session
+            .chain(_ => _.getValue<ISignInInfo>(SessionKey.SignInInfo))
+            .map(info => info[SignInInfoKeys.UserProfile])
+            .extract() as IUserProfile;
+
+        const appealsData = req.session
+            .chain(_ => _.getExtraData())
+            .chain(data => Maybe.fromNullable(data[AppealKeys.APPEALS_KEY]))
+            .extract() as Appeal;
 
+        try {
+            await this.emailService.send({
+                to: userProfile.email as string,
+                subject: 'Your appeal has been submitted',
+                body: {
+                    templateName: 'lfp-appeal-submission-confirmation',
+                    templateData: {
+                        companyNumber: appealsData[AppealKeys.PENALTY_IDENTIFIER][PenaltyIdentifierKeys.COMPANY_NUMBER],
```

Comment:
I believe in a past PR, you commented on the excessive use of keys? Not entirely sure what you said, but are we going to continue to actively use these keys? Isn't there a better way of doing it? We can probably look into it in another PR but im just mainly curious if there is a neater way of doing things.

Reply:
You are right, I am waiting for @franciscog-kainos to do it in separate PR. At this point in time I cannot do anything better than it without refactoring unrelated code which I am trying to avoid.